api/tests/mocha/resources.spec.js

```diff
@@ -0,0 +1,27 @@
+const rewire = require('rewire');
+const { expect } = require('chai');
+const path = require('path');
+
+let resources;
+
+describe('resources', () => {
+  beforeEach(() => {
+    resources = rewire('../../src/resources.js');
```

Comment:
Minor, but if we are not going to change anything between tests, is there any reason that we actually need `rewire`?

Reply:
Ahh, I see now. This file is just renamed from `environment.spec.js`, so you did not write it.  Well, no big deal either way if we change this or not...